mk/platform/linux.mk

```diff
@@ -35,3 +39,11 @@ endif
 MMAKE_KBUILD_ARGS_CONST := -C $(KPATH) NDEBUG=$(NDEBUG) GCOV=$(GCOV) CC=$(CC)
 MMAKE_KBUILD_ARGS = $(MMAKE_KBUILD_ARGS_CONST) $(MMAKE_KBUILD_ARGS_DBG)
 
+# Check if $(1) is greater than or equal to $(2)
+int_ge = $(shell test $(1) -ge $(2) 2>/dev/null && echo 1)
```

Comment:
What would be your opinion on using `sort` for the version comparison?
```
greater_version = $(shell printf "$(1)\n$(2)\n" | sort --version-sort | tail -1)

KBUILD_LIB_MAKE_TRG ?=
ifeq ($(call greater_version,$(LINUX_VERSION_3),6.0), $(LINUX_VERSION_3))
KBUILD_LIB_MAKE_TRG += $(lib_obj_path)
endif
```

It could eliminate `LINUX_VERSION_xyz` variables and hopefully contribute to readability?

Reply:
Looks nice, thank you.